.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ⁉️ Need help with Node.js?
```

Comment:
Is the leading `!?` intended? It looks like a typo.

Reply:
It is an emoji. See https://github.com/targos/node/issues/new/choose

Reply:
I don't see an emoji, I see the literal text `⁉️ Need help with Node.js?` on the choose site, same as the text above. 

I notice its the same at https://github.com/nodejs/node/issues/new/choose though, so its pre-existing, unrelated to this PR.